### ISSUE TITLE
Client: Clicking like highlights dislike

### ISSUE THREAD

**Issue by NathanAuckett** [gamejolt/next-issue-tracker#1200]
Using Game Jolt Client:

- Clicking the like button highlights the dislike button.
- Swapping back and forth between like and dislike increments the "likes" stat with each like making it count up infinitely.

Seems that the like button might actually be disliking the game. Clicking like, leaving the page, and going back to the game leaves the dislike button clicked and resets the like counter to what it was before you clicked the like button.
In short, the like button in Client seems to be disliking games.

**Comment by nilllzz** [comment]
Confirmed.

**Comment by nilllzz** [comment]
Fixed.
